package.json

```diff
@@ -2,7 +2,7 @@
   "name": "apache-daffodil-vscode",
   "displayName": "Apache Daffodil™ Extension for Visual Studio Code",
   "description": "Apache Daffodil™ Extension for Visual Studio Code providing DFDL syntax highlighting, DFDL code completion, DFDL schema debugging, and data editor",
-  "version": "1.3.1",
+  "version": "1.4.0-SNAPSHOT",
```

Comment:
This is interesting, and I agree with this change.  After we do a release, we should probably do this as the next merged commit, until the next release and so on.